### PR TITLE
[12.0][MIG] website_crm_recaptcha: Migration to v12

### PR DESCRIPTION
**Pull request by cristinamartinrod**
Depends on:

- [x] website_form_recaptcha - #572 

cc @Tecnativa

### CODE REVIEW

**Comment by yajo** [comment]
#572 is merged. I rebased, let's see if it goes :heavy_check_mark:.

File: website_crm_recaptcha/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2016-2017 LasLabs Inc.
+# Copyright 2019 Tecnativa - Cristina Martin R.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Website CRM - ReCaptcha",
+    "summary": 'Provides a ReCaptcha validation in Website Contact Form',
+    "version": "12.0.1.0.0",
+    "category": "Website",
+    "website": "https://laslabs.com/",
```
Comment (nikul-serpentcs) [inline website_crm_recaptcha/__manifest__.py:10]: `https://github.com/OCA/website`

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM code review

**Review by nikul-serpentcs — COMMENTED** [review]
minor change

**Review by pedrobaeza — COMMENTED** [review]
Please do README by fragments when doing the URL change.

**Comment by ska-ibees** [comment]
Can anyone please guide how to add recaptcha in login/signup form. Seems, record id is not matching with the model of the form. Thanks in advance.

```
<record id="auth_signup.model_res_partner" model="ir.model">
        <field name="website_form_recaptcha">True</field>
    </record>
```

**Comment by pedrobaeza** [comment]
@Yajo please check if everything is OK

**Comment by yajo** [comment]
Yes, I just updated the README. It seems to work fine.

**Review by yajo — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by ska-ibees** [comment]
> Can anyone please guide how to add recaptcha in login/signup form. Seems, record id is not matching with the model of the form. Thanks in advance.
> 

It seems that this module is meant for website and not for the /web. 
Could any one please put me in the right direction? I simply need to add recaptcha to the login and signup form.
Any suggestions please...!

**Comment by nikul-serpentcs** [comment]
@pedrobaeza @cristinamartinrod https://github.com/OCA/website/pull/573#discussion_r276601545 this comment is not attended!

**Comment by pedrobaeza** [comment]
@nikul-serpentcs please make a PR for changing it.

**Pull request merged**